### DISCUSSION TITLE
Version 1.7.19 seems to break build

### DISCUSSION THREAD

**Discussion by sebastianamaro** [tailwindlabs/headlessui#3102] [Help]
I have a tailwind template, which inside a Tab component, it has this code
    
  `<HeadlessTab as={Fragment}>
      {({ selected }) => (
        <li
          className={twMerge([
            "focus-visible:outline-none",
            fullWidth && "flex-1",
            list.variant == "tabs" && "-mb-px",
            className,
          ])}
          {...props}
        >
          <tabContext.Provider
            value={{
              selected: selected,
            }}
          >
            {typeof children === "function"
              ? children({
                  selected: selected,
                })
              : children}
          </tabContext.Provider>
        </li>
      )}
    </HeadlessTab>
`
    When building using the latest (1.7.19) version, it fails with this error
    
    src/components/Base/Headless/Tab/index.tsx:49:26 - error TS2345: Argument of type '{ selected: boolean; }' is not assignable to parameter of type 'TabRenderPropArg'.
  Property 'disabled' is missing in type '{ selected: boolean; }' but required in type 'TabRenderPropArg'.
  
  I cant seem to do any change that works. 
  
  This works with version 1.7.18

**Comment by RobinMalfait** [comment]
Can you share a minimal reproduction repo?

The `TabRenderPropArg` is the data that is exposed from within the component to the outside, e.g.:
```ts
<Tab>
{({ selected, disabled }) => {
  return //   ^^^^^^^^ is also exposed now
}}
</Tab>
```

It looks like you are trying to do some custom things and overriding internal information, e.g.: `children({ selected })`.

Is this from a component you wrote, or from some library you are using?

**Comment by antonioCrema** [comment]
Hello, has this problem been resolved?
